Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.cpp

```diff
@@ -408,32 +439,76 @@ namespace AzToolsFramework
 
     void QtEventToAzInputMapper::HandleKeyEvent(QKeyEvent* keyEvent)
     {
-        // Ignore key repeat events, they're unrelated to actual physical button presses.
+        const Qt::Key key = static_cast<Qt::Key>(keyEvent->key());
+        const QEvent::Type eventType = keyEvent->type();
+
+        // special handling for text events in edit mode
+        {
+            static Qt::Key lastKey = key;
```

Comment:
Would this be better as a member variable instead of a static? I doubt there will ever be multiple instances of QtEventToAzInputMapper in existence, but you never know.

Reply:
+1 - we generally want to avoid function level statics unless used for something like a singleton for initialization. This will break horribly if were ever decide to have another instance of `QtEventToAzInputMapper` which could potentially happen in future

Reply:
Sounds good, will promote to member variable.